plugin/trino-hive/src/test/java/io/trino/plugin/hive/containers/HiveMinioDataLake.java

```diff
@@ -61,15 +75,15 @@ public HiveMinioDataLake(String bucketName, Map<String, String> hiveHadoopFilesT
                                 .put("MINIO_SECRET_KEY", SECRET_KEY)
                                 .buildOrThrow())
                         .build());
-        this.hiveHadoop = closer.register(
-                HiveHadoop.builder()
-                        .withFilesToMount(ImmutableMap.<String, String>builder()
-                                .put("hive_minio_datalake/hive-core-site.xml", "/etc/hadoop/conf/core-site.xml")
-                                .putAll(hiveHadoopFilesToMount)
-                                .buildOrThrow())
-                        .withImage(hiveHadoopImage)
-                        .withNetwork(network)
-                        .build());
+
+        HiveHadoop.Builder hiveHadoopBuilder = HiveHadoop.builder()
+                .withImage(hiveHadoopImage)
+                .withNetwork(network);
+        Map<String, String> defaultHiveHadoopResourcesToMount = new HashMap<>(Map.of(
```

Comment:
```suggestion
                Map<String, String> defaultHiveHadoopResourcesToMount = ImmutableMap.<String, String>builder()
                .put("/etc/hadoop/conf/core-site.xml", getPathFromClassPathResource("hive_minio_datalake/hive-core-site.xml"))
                .putAll(hiveHadoopFilesToMount)
                .buildOrThrow();
```

Reply:
I had also tried that, however Guava's `ImmutableMap` doesn't allow duplicate keys to be inserted. Here we sometimes want to override the core-site.xml with a new one. Hence the standard `HashMap`.